### PR TITLE
migrate to latest scala version and update the compiler warnings to handle doobie macros

### LINKED ISSUE #17: Upgrade Scala 2.13.x
Now that the `frees` dependency has been removed, we can upgrade this project to the latest Scala version.

### PR DESCRIPTION
**Pull request by dmarticus**
Closes #17.

### CODE REVIEW

File: project/ProjectPlugin.scala [hunk 1]
```diff
@@ -34,6 +34,7 @@ object ProjectPlugin extends AutoPlugin {
       val paradise: String      = "2.1.1"
       val pureconfig: String    = "0.12.3"
       val scala212: String      = "2.12.10"
```
Comment (juanpedromoreno) [inline project/ProjectPlugin.scala:36]:
```suggestion
```

File: project/ProjectPlugin.scala [hunk 2]
```diff
@@ -169,8 +170,8 @@ object ProjectPlugin extends AutoPlugin {
     )
 
     lazy val noCrossCompilationLastScala: Seq[Def.Setting[_]] = Seq(
-      scalaVersion := V.scala212,
-      crossScalaVersions := Seq(V.scala212)
+      scalaVersion := V.scala213,
+      crossScalaVersions := Seq(V.scala213)
     )
 
     def isOlderScalaVersion(sv: String): Boolean =
```
Comment (juanpedromoreno) [inline project/ProjectPlugin.scala:177]: **suggestion**: This could be removed

File: project/ProjectPlugin.scala [hunk 3]
```diff
@@ -169,8 +170,8 @@ object ProjectPlugin extends AutoPlugin {
     )
 
     lazy val noCrossCompilationLastScala: Seq[Def.Setting[_]] = Seq(
```
Comment (juanpedromoreno) [inline project/ProjectPlugin.scala:172]: **suggestion**: As we are not cross-compiling with any other Scala version, we could remove these settings.

File: project/ProjectPlugin.scala [hunk 4]
```diff
@@ -196,9 +197,10 @@ object ProjectPlugin extends AutoPlugin {
         organizationHomePage = url("http://47deg.com"),
         organizationEmail = "hello@47deg.com"
       ),
-      scalaVersion := V.scala212,
-      crossScalaVersions := Seq(V.scala212), // , V.scala213), until next mu release
+      scalaVersion := V.scala213,
+      crossScalaVersions := Seq(V.scala213), // , V.scala213), until next mu release
```
Comment (juanpedromoreno) [inline project/ProjectPlugin.scala:201]:
```suggestion
      crossScalaVersions := Seq(V.scala213),
```

File: project/ProjectPlugin.scala (hunk 3 above)
Comment (dmarticus) [reply project/ProjectPlugin.scala:172]: yup, sounds good!

File: project/ProjectPlugin.scala (hunk 2 above)
Comment (dmarticus) [reply project/ProjectPlugin.scala:177]: will do

**Pull request merged**